templates/layout.html

```diff
@@ -181,11 +181,12 @@
                             <li><a href="{{ source_code }}">Source Code</a></li>
                             <li><a href="{{ source_code }}/blob/master/api.md">API</a></li>
                             <li><a href="/privacy">Terms & Privacy</a></li>
-                            <li><a href="/blog">Blog</a></li>
                             <li><a href="mailto:{{ support_mail }}">Support</a></li>
-                            {% if not irc_channel == "" %}
-                            <li><a href="{{ irc_channel }}" target="_blank">IRC</a></li>
+                            {% for key, value in support_channels.items() %}
+                            {% if not value == "" %}
```

Comment:
It's enough to simply check for the value
```suggestion
                            {% if value %}
```